README.md

```diff
@@ -45,9 +49,14 @@ optServiceName will default to `Entity Validation Service` if the field is not p
 
 All other fields must be provided.
 
+
+The businessVerificationCheck field allows the calling service to bypass the verification and
+continue to register where a successful match is found. By default "businessVerificationCheck" will be false.
+
 ```
 {
     "continueUrl" : "/test",
+    "businessVerificationCheck" : true ,
```

Comment:
The changes in this ticket are only for the Sole Trader endpoint so I don't think this should be included